src/javascripts/ng-admin/Crud/filter/maFilter.js

```diff
@@ -33,10 +32,15 @@ function maFilterDirective(FieldViewConfiguration) {
         scope: {
             filters: '=',
             datastore: '&',
-            values: '&'
+            values: '&',
```

Comment:
There is a mystery that I can't explain: `values` uses one-way databinding in an isolated scope, yet the watcher on `ListLayoutController` gets notified when the `values` change. Weird, but works.